admin/class-nginx-helper-admin.php

```diff
@@ -775,5 +778,17 @@ public function purge_all() {
 	public function display_notices() {
 		echo '<div class="updated"><p>' . esc_html__( 'Purge initiated', 'nginx-helper' ) . '</p></div>';
 	}
+	
+	/**
+	 * Determines if the current request is for importing Posts/ WordPress content.
+	 *
+	 * @return bool
+	 */
+	public function is_import_request() {
+		$import_query_var = sanitize_text_field( $_GET['import'] ?? '' );
```

Comment:
- `phpcs: Generic.Formatting.MultipleStatementAlignment.NotSameWarning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 1 space`

- `phpcs: WordPress.Security.NonceVerification.Recommended: Processing form data without nonce verification.`

- `phpcs: WordPress.Security.ValidatedSanitizedInput.MissingUnslash: $_GET['import'] not unslashed before sanitization. Use wp_unslash() or similar`